docs/4.3/enterprise/workflow/ssh_approval_slack.md

```diff
@@ -0,0 +1,223 @@
+  
+### Create Slack App
+
+We'll create a new Slack app and setup auth tokens and callback URLs, so that Slack knows how to notify the Teleport plugin when Approve / Deny buttons are clicked.
+
+You'll need to: 
+1. Create a new app, pick a name and select a workspace it belongs to. 
+2. Select “app features”: we'll enable interactivity and setup the callback URL here. 
+3. Add OAuth Scopes. This is required by Slack for the app to be installed — we'll only need a single scope to post messages to your Slack account. 
+4. Obtain OAuth token and callback signing secret for the Teleport plugin config. 
+
+#### Creating the app
+
+https://api.slack.com/apps 
+
+App Name: Teleport
+Development Slack Workspace: Pick the workspace you'd like the requests to show up in.
+
+![Create Slack App](https://p197.p4.n0.cdn.getcloudapp.com/items/llu4EL7e/Image+2020-01-09+at+10.40.39+AM.png?v=d9750e4fdc77901e0c2ffb2dc6040aee)
+
+#### Setup Interactive Components
+
+When someone tries approving / denying a request on Slack and clicks an Approve / Deny button, Slack will send a request to the Teleport Slack Plugin. In Slack App's settings, you must provide the URL for Slack to use for this. 
+
+This URL must match the URL setting in Teleport Plugin settings file (we'll cover that later), and be publicly accessible.
+
+For now, just think of the URL you'll use and set it in the Slack App's settings screen in Features > Interactive Components > Request URL.
+
+![Interactive Componets](https://p197.p4.n0.cdn.getcloudapp.com/items/v1umg7J0/Image+2020-01-09+at+10.48.08+AM.png?v=5e231f17304f506db13c3ccf445b6682)
+
+#### Selecting OAuth Scopes
+On the App screen, go to “OAuth and Permissions” under Features in the sidebar menu. Then scroll to Scopes, and add `chat.write` scope so that our plugin can post messages to your Slack channels.
+
+#### Add to Workspace
+
+![OAuth Token](https://p197.p4.n0.cdn.getcloudapp.com/items/E0uEg1ol/Image+2020-01-09+at+11.00.23+AM.png?v=1e28ff5bc4f7e0754acc9c7823f354a3)
+
+#### Obtain OAuth Token 
+
+![OAuth Token](images/oauth.png)
+
+#### Getting the secret signing token
+In the sidebar of the app screen, click on Basic. Scroll to App Credentials section, and grab the app's Signing Secret. We'll use it in the config file later.
+
+![Secret Signing Token](images/slackSigningSecret.png)
+
+## Installing the Teleport Slack Plugin
+To start using Teleport Plugins, you will need the teleport-slackbot executable.  See the [README](README.md) for building the teleport-slackbot executable in the Setup section.  Place the executable in the appropriate /usr/bin or /usr/local/bin on the server installation.
+
+### Configuration File
+Teleport Slack plugin has its own config file in TOML format. Before starting the plugin, you'll need to generate (or just copy the one below) and edit that config.
+
+To generate a config file, you can do this: 
+```bash
+teleport-slackbot configure > /etc/teleport-slackbot.toml
```

Comment:
Can't write directly to `/etc/`

Reply:
Yeah, I found that out https://github.com/gravitational/teleport-plugins/issues/90